templates/resources/virtual_cluster.md.tmpl

```diff
@@ -0,0 +1,49 @@
+---
+page_title: "{{.Name}} {{.Type}} - {{.ProviderName}}"
+subcategory: ""
+description: |-
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# {{.Name}} ({{.Type}})
+
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+
+## Example Usage
+
+
+### Virtual cluster deployment with host cluster and optional cluster group.
```

Comment:
```suggestion
### Virtual Cluster Deployment 
An example of a Palette Virtual Cluster with a host and optional cluster group.
```

We want to avoid long titles. We can keep the title short and add the explanation below.